build.sbt

```diff
@@ -94,10 +94,10 @@ lazy val core: CrossProject = crossProject.crossType(CrossType.Pure).
         println("CLDR files already available")
       }
     },
-    compile in Compile <<= (compile in Compile).dependsOn(downloadFromZip),
+    compile in Compile := (compile in Compile).dependsOn(downloadFromZip).value,
```

Comment:
the old `<<=` operator has been deprecated on `sbt 0.13.13`. I don't understand sbt well enough to tell why